addon/components/detail.js

```diff
@@ -83,22 +84,22 @@ export default Component.extend(PropTypeMixin, {
   },
 
   @readOnly
-  @computed('model')
+  @computed('reduxModel')
   renderModel (model) {
     return dereference(model || {}).schema
   },
 
   @readOnly
-  @computed('model', 'view')
+  @computed('renderModel', 'view')
   /**
    * Get the view to render (generate one if consumer doesn't supply a view)
    * @param {BunsenModel} model - the model schema to use to generate a view (if view is undefined)
    * @param {BunsenView} view - the view to use (if given)
    * @returns {BunsenView} the view to render
    */
   renderView (model, view) {
-    view = !_.isEmpty(view) ? view : getDefaultView(model)
-    return isEmberObject(view) ? view : recursiveObjectCreate(view)
+    view = !_.isEmpty(view) ? convertView(model, view) : getDefaultView(model)
```

Comment:
Does `convertView()` work with both POJO's and Ember Objects? If not then this is a regression as the API allows for the `view` property to be a POJO or an Ember Object.